src/utils/formatters.ts

```diff
@@ -0,0 +1,50 @@
+import { isArray } from 'lodash'
+import { IMiddleware } from '../types/interfaces'
+
+/**
+ * Works only with first middleware argument and prints his value if it consists of a primitive value array or an object containing such arrays
+ * @param {string} middlewareName
+ * @param {IMiddleware} middleware object containing all necessary information about actual middleware
+ * @return { string } middleware's description
+ * */
+export const basicArrayFormatter = (middlewareName: string, middleware: IMiddleware): string => {
+	const permissionsResult = `${middlewareName}: `
+
+	if (!middleware.isUsed) {
+		return `${permissionsResult}NO`
```

Comment:
@kot-1999 result for `!middleware.isUsed` could be united between this function (`basicArrayFormatter`) and `defaultFormatter` function

Reply:
@daniel-zavacky This are two different formatters. In case there will come some attributes  in a **middleware**, **basicArrayFormatter** will try to format them when the default one will only say if the middleware is used

Reply:
@LubomirIgonda1 Don't we want to use the same value in formatters for unused middlewares? Now in one case we have **middlewareName: NO** and in the second one it's **middlewareName: false**

Reply:
@kot-1999 it is a formatter, so it does not matter. It is just utils fn and can by modiffy by user as he wish